pages/1.12/cli/index.md

```diff
@@ -8,137 +8,63 @@ excerpt: Understanding the command line interface utility in DC/OS
+
+Commands:
+  auth
+      Authenticate to DC/OS cluster
+  cluster
+      Manage your DC/OS clusters
+  config
+      Manage the DC/OS configuration file
+  help
+      Help about any command
+  plugin
+      Manage CLI plugins
+
+Options:
+  --version
+      Print version information
+  -v, -vv
+      Output verbosity (verbose or very verbose)
+  -h, --help
+      Show usage help
+
+Use "dcos [command] --help" for more information about a command.
 ```
 
-# Displaying the DC/OS CLI version
+# Setting up a cluster
 
-To display the DC/OS CLI version, run:
-
-```
-dcos --version
-```
-
-
-<a name="configuration-files"></a>
-# DC/OS CLI versions and configuration files
-
-DC/OS CLI 0.4.x and 0.5.x use a different structure for the location of configuration files.
-
-DC/OS CLI 0.4.x has a single configuration file, which by default is stored in `~/.dcos/dcos.toml`. In DC/OS CLI 0.4.x you can optionally change the location of the configuration file using the [`DCOS_CONFIG`](#dcos-config) environment variable.
-
-DC/OS CLI 0.5.x has a configuration file for each connected cluster, which by default are stored in `~/.dcos/clusters/<cluster_id>/dcos.toml`. In DC/OS CLI 0.5.x you can optionally change the base portion (`~/.dcos`) of the configuration directory using the [`DCOS_DIR`](#dcos-cdir) environment variable.
-
-**Note:**
-- Updating to the DC/OS CLI 0.5.x and running any CLI command triggers conversion from the old to the new configuration structure.
-- After you call `dcos cluster setup`, (or after conversion has occurred), if you attempt to update the cluster configuration using a `dcos config set` command, the command prints a warning message saying the command is deprecated and cluster configuration state may now be corrupted.
-
-# Environment variables
-
-The DC/OS CLI supports the following environment variables, which can be set dynamically.
-
-<a name="dcos-cluster"></a>
-#### `DCOS_CLUSTER` (DC/OS CLI 0.5.x and higher only)
-
-The [attached](/1.10/cli/command-reference/dcos-cluster/dcos-cluster-attach/) cluster. To set the attached cluster, set the variable with the command:
-
-```bash
-export DCOS_CLUSTER=<cluster_name>
-```
-
-<a name="dcos-config"></a>
-#### `DCOS_CONFIG` (DC/OS CLI 0.4.x only)
-
-The path to a DC/OS configuration file. If you put the DC/OS configuration file in `/home/jdoe/config/dcos.toml`, set the variable with the command:
+In order to interact with your cluster, you first need to set up the CLI.
 
 ```bash
-export DCOS_CONFIG=/home/jdoe/config/dcos.toml
+$ dcos cluster setup <cluster-url>
 ```
 
-If you have the `DCOS_CONFIG` environment variable configured:
+After following the login flow, your CLI is now ready to interact with your cluster. You will notice that it now has additional commands such as `marathon`, `node`, `package` etc. These commands come from the [plugins](/1.12/cli/plugin), dcos-core-cli and, if applicable, dcos-enterprise-cli, which get automatically installed as part of the setup command.
```

Comment:
Rewrite "These commands come from the [plugins](/1.12/cli/plugin), dcos-core-cli and, if applicable, dcos-enterprise-cli, which get automatically installed as part of the setup command."

with 
These commands come from the [plugins](/1.12/cli/plugin), dcos-core-cli and, if applicable, dcos-enterprise-cli, which is automatically installed as part of the setup command.